src/linagora.esn.unifiedinbox/app/directives/main.js

```diff
@@ -387,19 +387,6 @@ angular.module('linagora.esn.unifiedinbox')
     };
   })
 
-  .directive('emailBodyEditor', function(emailBodyService) {
```

Comment:
Why is that removed ?

Reply:
Because it is not used anymore. And the template has been not exist. This belong to the commit that I explained.